src/appsettings/settings.py

```diff
@@ -837,12 +930,12 @@ def __init__(self, name='', default=None, required=False,
                 the setting's prefix (overrides ``AppSettings.Meta`` prefix).
             call_default (bool): whether to call the default (if callable).
             transform_default (bool): whether to transform the default value.
-            **checker_kwargs: keyword arguments passed to the checker.
+            validators (list of callables): list of additional validators to use.
+            min_length (int): minimum length of the iterable (included).
+            max_length (int): maximum length of the iterable (included).
+            empty (bool): whether empty iterable is allowed. Deprecated in favor of min_length.
```

Comment:
Do min_length and max_length make sense here? Maybe we should instead warn if they are present and pop them from the kwargs.

Reply:
Indeed, I don't see any point. That also applies to `empty` argument.